app/components/dashboard/run/left-panel/template.hbs

```diff
@@ -126,7 +126,7 @@
       <div class="ui form" style="margin-bottom:15px;">
           <div class="required inline field">
               <label>Please choose a target repository:</label>
-              {{#ui-dropdown class=repoDropdownClass allowAdditions=false selected=selectedRepository onChange=(action 'onRepositoryChange')
+              {{#ui-dropdown class=repoDropdownClass allowAdditions=false selected=selectedRepositoryName onChange=(action 'onRepositoryChange')
```

Comment:
Forgive me, but I'm not understanding the changes from ` selectedRepository` to `selectedRepositoryName` here.

If we used to select the entire repository object with the dropdown, then why change it to select only the name if we would then need to immediately do a lookup by name to get the full object? Am I just overlooking something obvious?